modules/compliance-rescan.adoc

```diff
@@ -11,6 +11,22 @@ Typically you will want to re-run a scan on a defined schedule, like every Monda
 $ oc annotate compliancescans/<scan_name> compliance.openshift.io/rescan=
 ----
 
+After rescan, four mc will be generated.
```

Comment:
This does not happen for all profiles.  It's only applicable to the rhcos-moderate profile.

Reply:
Should I explicitly mention there as such?

Reply:
Yes, that would be ideal.

Reply:
ok. Changed the sentence